packages/sessions/src/trackers/local.ts

```diff
@@ -235,7 +235,12 @@ export class LocalConfigTracker implements ConfigTracker, migrator.PresignedMigr
     const savePayload = this.savePayload({ payload })
     const saveNextConfig = this.saveWalletConfig({ config: args.nextConfig })
 
-    const recovered = await v2.signature.SignatureCoder.recover(decoded, payload, this.provider)
+    const recovered = await v2.signature.SignatureCoder.recover(
+      decoded,
+      payload,
+      this.provider,
+      'ignore'
```

Comment:
Why are we passing "ignore" here if we do have a provider? aren't we risking contaminating the local database of signatures?

I guess the alternative would be to miss data, which may be worse.